tools/test/travis-ci/functions.sh

```diff
@@ -52,17 +52,27 @@ set_status()
 #
 # Sources a pre-compiled GCC installation from AWS, installing the archive by
 #  extracting and prepending the executable directory to PATH.
+# Ccache is enabled for `arm-none-eabi-`.
 # 
 # Note: Expects 'deps_url' and 'deps_dir' to already be defined.
 # 
-_install_gcc()
+_install_gcc_and_ccache()
 {
+  # Enable Ccache in Travis
+  ccache -o compiler_check=content
+  ccache -M 1G
+  pushd /usr/lib/ccache
+  sudo ln -s ../../bin/ccache arm-none-eabi-gcc
+  sudo ln -s ../../bin/ccache arm-none-eabi-g++
+  export PATH="$PATH:/usr/lib/ccache"
```

Comment:
Use `export PATH="/usr/lib/ccache:$PATH"` to ensure ccache links are used (PATH lookup is first listed first used).